src/microbe_stage/PatchMapGenerator.cs

```diff
@@ -190,4 +205,14 @@ private static void TranslatePatchNames()
         _ = TranslationServer.Translate("PATCH_ICE_SHELF");
         _ = TranslationServer.Translate("PATCH_PANGONIAN_SEAFLOOR");
     }
+
+    private static LocalizedStringBuilder GetPatchLocalizedName(string name, string localizedBiome)
+    {
+        var localizedNames = new LocalizedStringBuilder();
+        localizedNames.Append(name);
+        localizedNames.Append(" ");
+        localizedNames.Append(new LocalizedString(localizedBiome));
```

Comment:
This part should be one entire localized string so that different languages can override this.
So this should be text like "PATCH_NAME" with the English translation having "{0} {1}" in it.

Reply:
I dont understand what  you mean

Reply:
you mean to make a single string and append it?

Reply:
`new LocalizedString("PATCH_NAME", name, new LocalizedString(localizedBiome))`
something like that. I think that'll work to make the order of the name parts work, and then there's no need to have the string builder here but can instead just directly return that new localized string from this method.

Reply:
In existing code it's used in for example `new LocalizedString("COMPOUND_FOOD_SOURCE", compound.Name);`

Reply:
now that i think about it, why would i do that since the only situation in the codebase the localized string is used with the string builder?

Reply:
using localizedStr.TroStr yields the translation key without the arguments, i think there needs to be used a localized Str builder nonetheles.

Reply:
String builder is only if you want to append multiple things. A single localized string can have a key (which is used to look up the text) and any number of string placeholder parameters. In this case like I showed (https://github.com/Revolutionary-Games/Thrive/pull/3118#discussion_r841255735) this can be implemented as a single LocalizedString.

Reply:
> using localizedStr.TroStr yields the translation key without the arguments, i think there needs to be used a localized Str builder nonetheles.

That can't be right. LocalizedString's ToString method should look up the translation and perform any string placeholder substitutions.

https://github.com/Revolutionary-Games/Thrive/blob/master/src/engine/LocalizedString.cs#L37